src/DistortableCollection.js

```diff
@@ -100,7 +101,16 @@ L.DistortableCollection = L.FeatureGroup.extend({
       /* conditional prevents disabled images from flickering multi-select mode */
       if (layer.editing.enabled()) {
         L.DomUtil.toggleClass(e.target, 'collected');
+        const newArr = arr.every((each) => {
+          return each._leaflet_id !== layer._leaflet_id;
+        });
+        if (newArr) {
+          arr.push(layer);
+        } else {
+          arr.splice(arr.indexOf(layer), 1);
+        }
       }
+      console.log(arr);
```

Comment:
Let's remove these once it's confirmed it's working, thanks!

Reply:
Alright @jywarren